pawsitive/config/database.yml

```diff
@@ -15,6 +15,10 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  host: localhost
+  port: 5432
+  username: gladdy
+  password: "12345"
```

Comment:
i know it's going to hard to remember, better to leave them empty.

Reply:
do you want me to just remove all of it or just lines 20 and 21

Reply:
is better to remove all.